feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -365,3 +388,169 @@ fun CaptureButton(
+                if (isDisabled) {
+                    Modifier
+                } else {
+                    Modifier.clickable {
+                        scope.launch {
+                            toggleState = when (toggleState) {
+                                ToggleState.Left -> ToggleState.Right
+                                ToggleState.Right -> ToggleState.Left
+                            }
+                            onToggleStateChanged(toggleState)
+                        }
+                    }
+                }
+            )
+    ) {
+        Box {
+            Row(
+                modifier = Modifier.matchParentSize(),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Box(
+                    Modifier
+                        .offset {
+                            IntOffset(
+                                x = with(density) {
+                                    animatedTogglePosition * (width - height).toPx()
+                                }.roundToInt(),
+                                y = 0
+                            )
+                        }
+                        .size(height)
+                        .clip(RoundedCornerShape(50))
+                        .background(circleColor)
+                )
+            }
+            Row(
+                modifier = Modifier.matchParentSize().then(
+                    if (isDisabled) Modifier.alpha(0.38f) else Modifier
+                ),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween
+            ) {
+                Icon(
+                    painter = leftIcon,
+                    contentDescription = "leftIcon",
+                    modifier = Modifier.padding(iconPadding),
+                    tint = if (isDisabled) {
+                        disableColor
+                    } else if (toggleState == ToggleState.Left) {
+                        iconSelectionColor
+                    } else {
+                        iconUnSelectionColor
+                    }
+                )
+                Icon(
+                    painter = rightIcon,
+                    contentDescription = "rightIcon",
+                    modifier = Modifier.padding(iconPadding),
+                    tint = if (isDisabled) {
+                        disableColor
+                    } else if (toggleState == ToggleState.Right) {
+                        iconSelectionColor
+                    } else {
+                        iconUnSelectionColor
+                    }
+                )
+            }
+        }
+    }
+}
+
+@Preview(backgroundColor = 0xFF000000, showBackground = true)
+@Composable
+private fun Preview_ToggleButton_Selecting_Left() {
+    val initialState = ToggleState.Left
+    var toggleState by remember {
+        mutableStateOf(initialState)
+    }
+    CompositionLocalProvider(LocalContentColor provides Color.White) {
```

Comment:
Rather than using this `CompositionLocalProvider`, you can instead wrap this with:
```
    MaterialTheme(colorScheme = darkColorScheme()) { }
```

Since we're likely to only want to use the dark color scheme for now. Alternatively, if you want to show both dark and light color schemes, you can replace the `colorScheme` argument with:
```
if (isSystemInDarkTheme()) darkColorScheme() else lightColorScheme()
```
and use multiple `@Preview` annotations on top of each of these functions:
```
@Preview(name = "Light Mode", uiMode = Configuration.UI_MODE_NIGHT_NO)
@Preview(name = "Dark Mode", uiMode = Configuration.UI_MODE_NIGHT_YES)
```
(it also helps to wrap the `ToggleButton` in a `Surface {}` block so the background adapts to the color scheme as well.)

Reply:
Thank you. Since I saw there is a method called "SettingsPreviewTheme" in the settings artifact, which has complete considerations for preview theme, I copied it and renamed it as "PreviewPreviewTheme" for reusing through component preview.